### DISCUSSION TITLE
Grafica assenze genitori/alunni

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#93] [Idee e proposte]
Buongiorno ,sarebbe possibile avere questa grafica (allego foto)per quanto riguarda la sezione assenze ,ritardi e uscite?Lo chiedo perché ho notato che il registro Giuaschool utilizza un sistema di sintesi ,ossia quando si effettua più di un'assenza consecutiva il registro fa così: assente dal 21 dicembre al 7 gennaio .Con questo sistema non si ha davanti un elenco giorno per giorno  e mese per mese delle assenze ,ritardi e uscite.Pertanto avrei pensato alla seguente grafica usata dal registro Axios qui sotto riportata 

https://user-images.githubusercontent.com/93823125/147840538-98af2727-f971-4348-938e-d10e33726c94.png

**Comment by trinko** [comment]
Al momento non sono previste modifiche alla grafica.
Lo segno per sviluppi futuri.
